src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/ForgotPasswordLinkSent.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿<div class="nhsuk-width-container">
+    <main class="nhsuk-main-wrapper">
+        <div data-test-id="login-page">
+            <div class="nhsuk-grid-row">
+                <div class="nhsuk-grid-column-two-thirds">
+
+                    <div data-test-id="go-back-link" class="nhsuk-back-link">
+                        <a class="nhsuk-back-link__link" asp-controller="Account" asp-action="Login">
+                            < Back to log in
```

Comment:
@kat1906 For the rendered `<` character in the front end does it use an SVG?

Reply:
yes

Reply:
but surely it doesn't matter if it looks and functions the same - in fact it might be preferable as it will always be the right size for the text (unless there is some sort of magic and it collapses into just the svg when on mobile view or something)